### DISCUSSION TITLE
Featuer request: surface the contents from `/LogFiles/eventlog.xml` in the portal

### DISCUSSION THREAD

**Discussion by nul800sebastiaan** [umbraco/Umbraco.Cloud.Issues#899] [Ideas]
### Issue description

This log file contains essential logging exceptions during site startup and should be easily available in the Cloud Portal.

I spent a lot of time looking into all kinds of different logs on the portal and they were not only far away from what I needed, they actively led me down the wrong path. If I could have found this log file easily, I would not have had 24 hours of down time (instead my problem was fixed in 20 minutes after I found the relevant error 😅). 

If I was a customer and not an HQ member, I would have had to have gone through support as well, but with the logs in the portal it would've been easy self-service and no need to involve support staff.

**Comment by jemayn** [comment]
Thanks for your suggestion @nul800sebastiaan, sorry we haven't gotten back to you sooner!

I can see this is a duplicate of https://github.com/umbraco/Umbraco.Cloud.Issues/discussions/833, so I will close this discussion as a duplicate and any discussions / votes can go in the other one 🙂
